app/controllers/statements_controller.rb

```diff
@@ -13,22 +13,17 @@ def initialize
     super
   end
 
-  # GET /accounts/:account_id/facilities/:facility_id/statements/:id
+  # GET /accounts/:id/statements
+  def index
+    @statements = @account.statements.paginate(page: params[:page])
```

Comment:
Just found an interesting behavior. Because `Account` `has_many :statements, through: :order_details` rather than being a direct relation, I'm seeing duplicate statements, one for each order detail on the statement.

Making it a direct relationship based on `Statement#account_id` seems like the right choice, but I'm not sure what effect that'll have on other places in the app. Maybe try it out and see if any specs break?

Other option is to do a `uniq`/`distinct` on the result.